docs/import-assistant.md

```diff
@@ -0,0 +1,67 @@
+ # Import Assistant
+ 
+Run powerful AI-enabled commands to assist with your import script development.
+
+&nbsp;
+### Install
+Add an npm script entry to your Edge Delivery project's `package.json`:
+
+```
+"assistant": "aem-import-helper assistant"
+```
+
+### Options
+
+- `--url`: The URL of the page to analyze
+- `--outputPath`: The directory to save the import scripts
+- `--name`: The name of the block
+- `--prompt`: Descriptive text to help understand the content
+
+
+&nbsp;
+## Commands
+
+
+### Start
+
+Start a new import project.
+
+##### Example:
+```npm run assistant -- start --url https://example.com --outputPath tools/importer```
+
+&nbsp;
+### Cleanup
+
+Add elements that can be removed from the document.
+#### Example:
+
+```npm run assistant --cleanup --url https://example.com --prompt "content to remove" --outputPath tools/importer```
```

Comment:
Need a better prompt example.  Use something real so people get the idea.  Something like:
"breadcrumbs and login dialog"
Maybe @arumsey has a better example.